project_issue_code/models/project_issue.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+
+
+class ProjectIssue(models.Model):
+    _inherit = 'project.issue'
+
+    issue_code = fields.Char(
+        string='Issue Code', required=True, default="/", readonly=True)
+
+    _sql_constraints = [
+        ('project_issue_unique_code', 'UNIQUE (issue_code)',
+         _('The code must be unique!')),
+    ]
+
+    @api.model
+    def create(self, vals):
+        if vals.get('issue_code', '/') == '/':
+            vals['issue_code'] = self.env['ir.sequence'].next_by_code(
+                'project.issue') or '/'
+        return super(ProjectIssue, self).create(vals)
+
+    @api.multi
+    def copy(self, default=None):
```

Comment:
I think you should put `copy=False` on the field, and let the `create` method manage this part, instead of duplicating code.
Moreover, this prevents customizing the code while duplicating a record.